jump.lua

```diff
@@ -4,8 +4,9 @@ jumpdrive.simulate_jump = function(pos, player, show_marker)
 
 	local targetPos = jumpdrive.get_meta_pos(pos)
 
-	if jumpdrive.check_mapgen(pos) then
-		return false, "Error: mapgen was active in this area, please try again later for your own safety!"
+	local mapgen_distance = jumpdrive.check_mapgen(pos)
+	if mapgen_distance then
+		return false, "Error: mapgen was active "..math.floor(mapgen_distance).." / 200 nodes away, please try again later for your own safety!"
```

Comment:
```suggestion
		return false, "Error: mapgen was active "..math.floor(mapgen_distance)
			.. " / 200 nodes away, please try again later for your own safety!"
```
I think this would be last one, also not too important (error from luacheck linter, max line length > 120).

Reply:
should be done